what_the_fec/templates/includes/insert_fieldset.html

```diff
@@ -0,0 +1,81 @@
+<div id="insert" style="display: block">
+    <form id="insert_{{table_name}}" action="/{{table_name}}" method="post">
+            <fieldset>
+                <legend><strong>Add item to {{table_name}}</strong></legend>
+
+                {% for col_name in items[0].keys() %}
```

Comment:
this whole file is really awesome. great work.

Reply:
Woot, thanks!